README.md

```diff
@@ -1,6 +1,8 @@
 # Timescale Terraform Provider
 The Terraform provider for [Timescale](https://www.timescale.com/cloud).
 
+Does not work for Managed Service for TimescaleDB.
```

Comment:
```suggestion
Does not work for Managed Service for TimescaleDB.
```
```suggestion
Does not work for Managed Service for Azure or GCP.
```

or maybe even Terraform only supports Managed Timescale for AWS?

Reply:
it's all aiven proxies basically ( aws included ) but i'm not certain customers know which category they fall in 🤔

Reply:
true I was just going off the naming on the landing pages.